explore/shortcuts.md

```diff
@@ -10,17 +10,17 @@ Shortcuts are common tasks that can be triggered by keyboard combinations or con
 
 ## Registering with a Canvas
 
-There are many standard shortcuts defined (such as `fyne.ShortcutCopy`) which are connected to standard keyboard shortcuts and right-click menus. The first step to adding a new `Shortcut` is to define the shortcut. For most uses this will be a keyboard triggered shortcut, which is a desktop extension. To do this we use `desktop.CustomShortcut`, for example to use the Tab key and Control modifier you might do the following:
+There are many standard shortcuts defined (such as `fyne.ShortcutCopy`) which are connected to standard keyboard shortcuts and right-click menus. The first step to adding a new `Shortcut` is to define the shortcut. For most uses this will be a keyboard triggered shortcut, which is a desktop extension. To do this we use `fyne.KeyModifierControl`, for example to use the Tab key and Control modifier you might do the following:
```

Comment:
This change does not seem right - it is still using `CustomShortcut`

Reply:
What’s the alternative? The one above doesn’t seem to be deprecated: https://pkg.go.dev/fyne.io/fyne/v2/driver/desktop#CustomShortcut.

Reply:
Yes. What I mean is that I think this documentation was correct before the change.

Reply:
Ah, I see what you mean now. Will get that fixed.

Reply:
Should be fixed now. Sorry about that.